security/guard_authentication.rst

```diff
@@ -221,27 +221,27 @@ This requires you to implement several methods::
             return null;
         }
 
-        public function onAuthenticationFailure(Request $request, AuthenticationException $exception)
+        public function onAuthenticationFailure(Request $request, AuthenticationException $exception): JsonResponse
```

Comment:
As far as I know the documentation, return type declarations are not used.
@javiereguiluz Can you give an opinion on this?